AirCasting/CreateSessionViews/SelectDeviceView.swift

```diff
@@ -108,6 +108,7 @@ struct SelectDeviceView: View {
             micLabels
         })
         .buttonStyle(WhiteSelectingButtonStyle(isSelected: selected == 2))
+        .disabled(microphoneManager.isRecording)
```

Comment:
Maybe we could add and alert that can inform user about the thing that he can only make only one session and that's why it is grayed out?